slamdog:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  joint_left_controller:
      type: velocity_controllers/JointVelocityController
      joint: joint_left_wheel
      # pid: {p: 100.0, i: 0.01, d: 10.0}

  joint_right_controller:
      type: velocity_controllers/JointVelocityController
      joint: joint_right_wheel
      # pid: {p: 100.0, i: 0.01, d: 10.0}
